plugins/org.python.pydev/src/org/python/pydev/builder/flake8/Flake8PrefPage.java

```diff
@@ -84,25 +101,45 @@ public void widgetSelected(SelectionEvent e) {
                 true, parent);
         addField(fileField);
 
-        addField(new RadioGroupFieldEditor(Flake8Preferences.FLAKE8_E_SEVERITY, "E Severity", COLS, LABEL_AND_VALUE,
-                parent, true));
+        jsonFieldEditor = new JsonFieldEditor(Flake8Preferences.FLAKE8_CODES_CONFIG,
+                WrapAndCaseUtils.wrap("desc", 90),
```

Comment:
A proper description needs to be added here.

It can be something as:

`"Flake 8 severity map (unmatched entries will be marked as \"warning\")"`